### ISSUE TITLE
Console Hyperlink to Source

### ISSUE THREAD

**Issue by jeffhoye** [GoClipse/goclipse#196]
This is a feature request, but I think it may be easy to impl.  

Please add source hyperlinking support in the console to the source.  

The idea is that you can click on a stack dump or build error and it brings you to the line of the source file.

Example in console:
`
	c:/go/src/database/sql/sql.go:589 +0x53  
`

Clicking on that opens source on line 589 of sql.go

Here's the how-to:
http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fextension-points%2Forg_eclipse_ui_console_consolePatternMatchListeners.html

I'm happy to eventually contribute this feature, but I'm currently extremely busy and haven't built an eclipse addin before.  Therefore, I'm just putting the bug in your ear now in case you want to get to it before me.  :)  Thx.

**Comment by bruno-medeiros** [comment]
Yeah, I'm busy with other features, so it's not a priority for me. Also, there is the Problems view, which you can use to navigate to the file with the underlying problem, which I think is a more natural way to do things. 

The default Problems view shows all problems in the workspace though. But it is possible to create a new Problems view, that only shows the problems in, say, the current project.
